src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.html

```diff
@@ -16,9 +16,12 @@
     >
       Currently Revising: {{ learningObject?.name }}
     </div>
-    <div class="top__left">
+    <div class="top__left" *ngIf="!adminMode">
```

Comment:
I noticed in the video the div took a moment to toggle which is fine. One way to make this more dynamic is to use a `ng-container` and toggle between templates. If you're willing, I can walk you through it otherwise it's no big deal